client/components/thank-you/index.tsx

```diff
@@ -34,12 +34,27 @@ const ThankYouContainer = styled.div< ThankYouThemeProps >`
 	}
 `;
 
-const ThankYouHeader = styled.div`
+const ThankYouHeader = styled.div< ThankYouThemeProps >`
 	width: 100%;
-	height: 240px;
-	background-color: var( --studio-gray-0 );
 	display: flex;
 	justify-content: center;
+	background-color: ${ ( props ) => props.theme.colors.backgroundColorHeader };
+	height: 352px;
+	img {
+		width: auto;
+		height: 200px;
+		margin-bottom: 14px;
+	}
+`;
+
+const ThankYouFooterContainer = styled.div< ThankYouThemeProps >`
+	h1 {
+		font-size: ${ ( { subtitle = false } ) => ( subtitle ? '1.5em' : '2em' ) };
```

Comment:
I think we can remove the `subtitle` argument handling -- this `h1` is always going to be `2em` AFAICT.
```suggestion
		font-size: 2em;
```